### PR TITLE
Clean up wording in README

### PR DESCRIPTION
**Pull request by ErikBoesen**
I need to proofread...

### CODE REVIEW

**Review by JosephCarpenter — DISMISSED** [review]
Can't edit individual lines in the paragraph so I'll put changes here:
 
- beginning: ...keeping track of time has been challenging for me ('challenged in' sounds off)
- switches between first and second person
- end: ...*notice* certain events there

**Review by JosephCarpenter — COMMENTED** [review]
"moved to your wrist" 
so close...

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Wristcoach helps out by giving a sophisticated and configurable experience which
...
+As drive coach, keeping track of time is challenging. Depending on one's station, it can be tough to see the countdown on the other side of the field, and looking to the livestream screen is time-consuming when it's even possible. Furthermore, it's difficult to pay enough attention to any clock to remember exactly when drivers must prepare for endgame or other actions. Thus, a simple smartwatch app makes menial mental tasks a thing of the past. With robotic timing tasks moved to your wrist, coaches can focus on the field as they should.
```
Comment (JosephCarpenter) [inline README.md]: also you may want to make that "As a drive coach"

**Comment by ErikBoesen** [comment]
I hope GitHub adds a cry emoji

**Review by JosephCarpenter — APPROVED** [review]
Still should probably be 'as a drive coach' but I don't want to keep beating a dead horse here

**Pull request merged**